File: questao_3/main.py
import random


def shuffle(arr):
    size = len(arr)
    for i in range(0, size-1, 1):
        j = random.randint(0, i)
        arr[i], arr[j] = arr[j], arr[i]

    return arr


def main():
    arr_test = ["Joao", "Maria", "Jose"]
    print(shuffle(arr_test))


if __name__ == "__main__":
    main()

File: questao_3/main.js
const shuffle = (arr) => {
  const size = arr.length;
  for (let i = 0; i < size - 1; i++) {
    const j = i + Math.floor(Math.random() * (size - i));

    [arr[i], arr[j]] = [arr[j], arr[i]];
  }
  return arr;
};

const testArray = ["John", "Mary", "Jake"];
